### DISCUSSION TITLE
Display issues with page TOCs in edit view

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#820] [0. General bugs and questions]
So this is only in edit view, not in the published docs, but bringing it up to make sure that it doesn't migrate to the published view.

In edit view, there are a couple of display issues with the page-level table of contents at the right.

![image](https://github.com/user-attachments/assets/2a6c31b1-107a-4e19-a083-f7364b8ca1f1)

Items at the same level don't always line up. For example, **Viewing and copying your organization identifier** and **Configuring notifications for column comments** are actually both Heading 1, but the second heading is a little bit indented.

For sections with longer titles, subsequent lines are indented way too much - the second line starts almost in the middle of the contents column. This makes it very difficult to read - the eye has to go looking for the text. In the published docs, the subsequent lines do not indent.

**Comment by gregberge** [comment]
Hi @JaniceManwiller, thanks for reporting it. A fix is being deployed.
